src/model/draft/mef-legato-services.yang

```diff
@@ -630,7 +630,7 @@ module mef-legato-services {
                                 "mef-global:l2cp-cos-id]/" +
                                 "mef-global:map-entries/" +
                                 "mef-global:map-entry" +
-                                  "[mef-global:cos-name]" +
+                                "[mef-global:cos-name]" +
```

Comment:
Not clear on what rules you are using to determine indentation here. But I have revered the change.